pom.xml

```diff
@@ -70,6 +70,11 @@
 				<artifactId>aws-java-sdk-core</artifactId>
 				<version>1.11.52</version>
 			</dependency>
+			<dependency>
+				<groupId>com.amazonaws</groupId>
+				<artifactId>aws-java-sdk-s3</artifactId>
+				<version>1.11.52</version>
```

Comment:
Probably worth extracting this to a property now that it is used in multiple places

Reply:
:+1:

Reply:
Okay, I'll probably update it too and run through the unit tests. I do have some (unrelated to this) that fail locally, is that expected or am I missing some set up?

Reply:
Right now the build is green so I would not expect any failures locally.  Also CircleCI built your branch fine as well

Reply:
fixed